README.md

```diff
@@ -1,22 +1,35 @@
 # ClubHouse Pull Request Github Action
 
-Insert description here, Rae.
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+![Test](https://github.com/farmersdog/clubhouse-pr/workflows/Test/badge.svg)
+
+Automatically update your Github Pull Request with a Clubhouse title in the format: `(feat) Some Feature [ch123]`.
 
 ## Inputs
 
 ### `ghToken`
 
 **Required** GITHUB_TOKEN
 
+### `chToken`
+
+**Required** Clubhouse API Token
+
 ## Outputs
 
-### `pr-title`
+### `prTitle`
 
 The title of the pull request
 
+## Development
+
+Run `yarn tdd` to watch Jest tests as you make your changes.
+
+Run `yarn lint:watch` to watch for ESLint errors/warnings.
+
 ## Example usage
 
-Note: This is for use when *opening* or *synchronizing* a pull request.
+Note: This is for use when _opening_/synchronizing a pull request.
```

Comment:
Just to confirm, is there some kind of guard against this re-adding the same story id over and over to the PR Title if we run on every synchronize? Sorry if im missing below!

Reply:
So, we're gonna only use it on "open" for now actually!